### PR TITLE
Add flag for save button

### PR DESCRIPTION
**Pull request by bjfletcher**
For the new stream list.

Please note that this branches off #1179

### CODE REVIEW

**Comment by bjfletcher** [comment]
Green build: https://circleci.com/gh/Financial-Times/next-stream-page/3285

File: bower.json [hunk 1]
```diff
@@ -3,20 +3,22 @@
...
+    "o-date": "^2.4.0",
     "o-fetch-jsonp": "^1.0.1",
     "o-icons": "^4.1.2",
     "o-share": "^2.1.3",
     "o-video": ">=0.1.0 <1"
   },
   "resolutions": {
-    "n-card": "^5.4.0",
+    "o-date": "^2.4.0",
+    "n-card": "^5.5.0",
```
Comment (ironsidevsquincy) [inline bower.json]: why are these needed?

File: n-makefile.json [hunk 2]
```diff
@@ -13,12 +13,14 @@
...
-      "@financial-times/n-section",
       "@financial-times/n-image",
+      "@financial-times/n-section",
       "@financial-times/n-ui",
+      "n-card",
       "n-image",
       "n-section",
-      "n-ui"
+      "n-ui",
+      "o-date"
```
Comment (ironsidevsquincy) [inline n-makefile.json]: shouldn't need any of these

Comment (bjfletcher) [inline n-makefile.json]: `o-date` API is used for formatting the date server-side before it goes to client side (so you don't see any change as `o-date` responsive stuff loads)

File: bower.json (hunk 1 above)
Comment (bjfletcher) [inline bower.json]: no longer needed - removed now

File: n-makefile.json (hunk 2 above)
Comment (ironsidevsquincy) [inline n-makefile.json]: Hmmm, we already do that in n-card, and it isn't needed then - https://github.com/Financial-Times/n-card/blob/master/components/timestamp/timestamp.js

Also, like n-card, shouldn't that formatting happen client side? As could have any old time on the server (it fades in the time so it's less jarring)
